girder/utility/server.py

```diff
@@ -76,7 +75,8 @@ def _configureStaticRoutes(webroot, plugins, event=None):
     webroot.updateHtmlVars({
         'apiRoot': config.getConfig()['server']['api_root'],
         'staticRoot': staticRoot,
-        'plugins': plugins
+        'plugins': plugins,
+        'title': model_importer.ModelImporter.model('setting').get(constants.SettingKey.BRAND_NAME)
```

Comment:
I don't think this needs to be here. The Webroot will be instantiated with the correct value.